.github/workflows/integration.yml

```diff
@@ -37,6 +37,8 @@ jobs:
             ARGS: MYSQL_VER=mariadb:10.6.3 ES_VER=opensearchproject/opensearch:1.0.0
           - TEST_NAME: "HTTPS"
             ARGS: TEST_HTTPS=Yes
+          - TEST_NAME: "Force JEncode"
+            ARGS: DIRAC_USE_JSON_ENCODE=Yes DIRAC_USE_JSON_DECODE=Yes
```

Comment:
...just to say that this was the way to go! I don't know why we didn't think to do this before!